src/atlas/projection/detail/ProjectionUtilities.h

```diff
@@ -0,0 +1,150 @@
+/*
+ * (C) Copyright 2020 UCAR
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
```

Comment:
Possibly rename ProjectionUtilites.h to SphereProjectionUtilities.h ?

Reply:
Do we have non-sphere projections? My guess is that Willem will want to do something different with this anyway.